testbots/Directory.Build.props

```diff
@@ -10,6 +10,10 @@
     <NoWarn>$(NoWarn);SA0001;CS1573;CS1591;CS1712;SX1309</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

Comment:
Let's keep explicit references to newtonsoft on each project, Muthu is working on trying to see if we can remove this dependency for v5